packages/jest-validate/src/types.ts

```diff
@@ -11,7 +11,9 @@ type Title = {
   warning?: string;
 };
 
-export type DeprecatedOptions = Record<string, Function>;
+export type DeprecatedOptionFunc = (arg: Record<string, any>) => string;
```

Comment:
```suggestion
export type DeprecatedOptionFunc = (arg: Record<string, unknown>) => string;
```
?

Reply:
iirc no because then TS knows too much and would force us to properly type the deprecated types too - I think a good way to make this change would be to do it after the latest deprecated options are removed, so to require all future deprecated options to be more accurately typed.

Reply:
sounds good 👍